### PR TITLE
Ajout de 10 fiches fr - en

### PR DESCRIPTION
**Pull request by webafrancois**
Moodle, Mumbleserver, Kanboard, Keeweb, Peertube, Phpmyadmin, Phpsysinfo, searx, shellinabox, unattended-upgrades

### CODE REVIEW

File: app_searx_fr.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+Searx est un métamoteur de recherche libre et décentralisé. Les requêtes effectuées ne sont pas enregistrées par searx afin de protéger la vie privée des utilisateur·rice·s.
+En tant que métamoteur, searx collecte et affiche les résultats issus de plusieurs dizaines de moteurs de recherche. Les moteurs utilisés (ou non) sont paramétrables dans les préférences.
+En tant que logiciel décentralisé, searx est installé sur différentes instances (différents serveurs). En France, des associations comme Framasoft, Aquilenet ou La Quadrature du Net l'ont installé sur leurs serveurs.
+
+## Liens utiles
+
++ Site web : [searx.me](https://searx.me/)
++ Documentation officielle : [asciimoo.github.io/searx](https://asciimoo.github.io/searx/)
++ Démonstration: [Démo](https://demo.yunohost.org/searx/)
```
Comment (Plumf) [inline app_searx_fr.md:17]:
```suggestion
+ Démonstration : [Démo](https://demo.yunohost.org/searx/)
```

File: app_phpsysinfo_fr.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+[![Install phpsysinfo with YunoHost](https://install-app.yunohost.org/install-with-yunohost.png)](https://install-app.yunohost.org/?app=phpsysinfo) [![Integration level](https://dash.yunohost.org/integration/phpsysinfo.svg)](https://dash.yunohost.org/appci/app/phpsysinfo)
+
+### Index
+
+Un script PHP personnalisable qui affiche bien les informations sur votre système (CPU, Ram, disques...).
+
+## Liens utiles
+
++ Site web : [phpsysinfo.github.io/phpsysinfo](https://phpsysinfo.github.io/phpsysinfo/)
++ Démonstration: [Démo](https://phpsysinfo.github.io/phpsysinfo/demos.html)
```
Comment (Plumf) [inline app_phpsysinfo_fr.md:12]:
```suggestion
+ Démonstration : [Démo](https://phpsysinfo.github.io/phpsysinfo/demos.html)
```

Comment (Plumf) [reply app_phpsysinfo_fr.md:12]: Thanks for your works, LGTM :+1:

**Comment by Psycojoker** [comment]
Merci beaucoup :heart:

**Pull request merged**